day_6/exercises/burrito.rb

```diff
@@ -11,9 +11,28 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+
+  def add_topping(new_topping)
+    toppings.push(new_topping)
+  end
+
+  def remove_topping(old_topping)
```

Comment:
What would be a better name for the parameter here?

Reply:
Sorry, I clicked "Resolve conversation" thinking it would take me to what you were asking me to resolve.  I'll look at it and click "Resolve conversation" when it's actually resolved!